mocks/pkg/fixtures/Fooer.go

```diff
@@ -19,6 +19,7 @@ func (_m *Fooer) Baz(path string) func(string) string {
 	ret := _m.Called(path)
 
 	var r0 func(string) string
+
```

Comment:
Could we get rid of whatever change causes this whitespace?

Reply:
Sure, no problem